sys/zuse.hoon

```diff
@@ -6882,28 +6882,33 @@
       ::                                                ::  ++lef:yu:chrono:
       ++  lef                                           ::  leapsecond dates
         ^-  (list @da)
-        :~  ~2015.6.30..23.59.59   ~2012.6.30..23.59.59
```

Comment:
One minor comment: For better diff legibility around things like this in the future, we should probably have one entry per line